docs/excel/make-custom-functions-compatible-with-xll-udf.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Make your custom functions compatible with XLL user-defined functions
+description: 'Enable compatibility with Excel XLL user-defined functions that have equivalent functionality to your custom functions'
+ms.date: 04/22/2019
+localization_priority: Normal
+---
+
+# Make your custom functions compatible with XLL user-defined functions
+
+If you have existing Excel XLLs, you can build equivalent custom functions in an Office web add-in to extend your solution features to other platforms such as online or macOS. However, Office web add-ins don't have all of the functionality available in XLLs. Depending on the functionality your solution uses, the XLL may provide a better experience than the web add-in custom functions on Excel for Windows.
+
+You can configure your Office web add-in so that when an equivalent XLL is already installed on the user's computer, Excel runs the XLL instead of your web add-in custom functions. The XLL is called equivalent because Excel will seamlessly transition between the XLL and the web add-in custom functions depending on which is installed on Windows.
```

Comment:
"your web add-in custom functions" -> "your web add-in's custom functions"
(x2 in this line)